dt/HardProv/Redfish/Hardware_Provisioning_Redfish.md

```diff
@@ -0,0 +1,51 @@
+# RHOSO Deployed Topology HardProv-3cont_2comp_2ironic-ipv4-geneve-redfish
+
+**Redfish / Sushy**
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 2023-12-18      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Compute nodes                                 | vm    | 2      |
```

Comment:
We really just need the 3-node Openshfit master/workr compo-node + 2 baremetal or virtual baremetal machines.